packages/mobile/src/exchange/ExchangeConfirmationCard.tsx

```diff
@@ -42,7 +33,7 @@ const getExchangeRate = (props: Props) => {
   }
 
   // For feed drilldown, the exchange rate has not been provided
-  return makerAmount.dividedBy(takerAmount)
+  return new BigNumber(makerAmount.amount).dividedBy(takerAmount.amount)
```

Comment:
Per the thread you started on Slack, would be cool if we could encapsulate more logic in MoneyAmount and avoid this pattern :)

Reply:
Yes we can definitely explore that.
I like the simplicity of having plain objects and functions operating on them. But money amount is an essential block in our app and we might benefit encapsulating it in its own class 👍